modules/ROOT/pages/manage-audit-logs.adoc

```diff
@@ -0,0 +1,292 @@
+
+Below you can find the minimal configuration required in your bootstrap config file to enable audit logging.
+
+[source, json, indent=0]
+----
+
+"logging": {
+    "audit": {
+	  “audit_log_file_path”: “/path/to/file.log”,
+    “enabled”: true,
+    }
+}
+
+----
+
+For further bootstrap configuration options, see xref:configuration-schema-bootstrap.adoc[].
+
+NOTE: Configuration via the bootstrap config will affect your configuration globally.
+
+To successfully configure audit logging, you must set a file path for your audit logs and set the `enabled` parameter to `true`.
+Enabling and disabling audit logging is handled by the Admin REST API at a cluster and Sync Gateway database level.
+Setting enabled to true without any specified enabled events will only enable events with `DefaultEnabled` set to `true`. 
+To view the list of audit events, see xref:audit-log-events.adoc[]
+
+=== Configuration with the Database Config
+
+You can alter the configuration per Sync Gateway database by changing the database config.
+
+Below is an example of a simple database configuration.
+
+[source, json]
+----
+{
+  "bucket": "todo", 
+  "num_index_replicas": 0,
+  "logging": {
+               "audit": {
+                          "enabled": false,
+                          "enabled_events": [],
+                          "disabled_users": [
+                                              {
+                                                "name": "user1",
+                                                "domain": "sgw"
+                                              }
+                                            ]
+                        },
+  "disabled_roles": [
+                      {
+                        "name": "role1",
+                        "domain": "sgw"
+                      }
+                    ]
+               }              
+}  
+----
+
+For further database configuration options, see xref:configuration-schema-database.adoc[].
+
+IMPORTANT: If you enable audit logging in your bootstrap config it is enabled globally.
+Conversely, it is important to note that if audit logging is enabled in your database config but *not* your bootstrap config, no audit logging will occur.
+
+Couchbase recommends you only use configuration at the database level if you wish to alter your settings per database. 
+
+== Filtering Audit Logs
+
+You can filter audit logs to customize events that are stored in the log file. 
+You can alter the following to affect events stored in the log file:
+
+* Enabling and disabling users - This affects all events related to a specified user.
+If a given user is disabled, no audit events from the user will be logged.
+
+* Enabling and disabling events - This affects the specified events globally.
+If a given event is disabled, it will not be logged regardless of the user.
+
+NOTE: If an audit event has `DefaultEnabled` set to `true`, it does not need to be explicitly listed in the `enabled_events: []` array.
```

Comment:
I don't think this is correct.  If enabled_events is specified in the bootstrap config, that overrides the default set of enabled events.  Only if enabled_events is omitted will the default set of optional events be enabled.
https://confluence.issues.couchbase.com/wiki/spaces/SWD/pages/2387378237/SGW+3.2.0+Audit+Logging#Bootstrap%2Fstartup-config